Fix Pagination Alignment

MondalSayantan opened the pull request:
- Remove 'row' class to prevent overflow on mobile devices
- Aligned pagination component with the rest of the body
- Verified that removing class does not have any other effect to the UI in different screens
- Basic Code formatting done

Before
![image](https://user-images.githubusercontent.com/62999423/215829874-0ddbc895-642b-483f-ac70-424547b767e7.png)

Now
![image](https://user-images.githubusercontent.com/62999423/215829968-0d9456a1-5e4a-4ac4-9895-4838725a0c9a.png)

MondalSayantan committed:
Refactor Pagination CSS:

- Remove 'row' class to prevent overflow on mobile devices
- Aligned pagination component with the rest of the body

MondalSayantan committed:
Refactor Pagination CSS

- Reverted some linting changes which might be causing issues

zbynek commented:
Please revert the unrelated changes (whitespace and brackets), otherwise looks good, thanks.

MondalSayantan committed:
Refactor Pagination CSS

- Reverted whitespaces changes made in previous commit
- Removed 'row' class from div to prevent overflow in mobile devices

MondalSayantan commented:
Reverted all unrelated changes in the latest commit. Thanks for your review

zbynek committed:
Add newline
